src/components/NewSnippet.jsx

```diff
@@ -1,10 +1,60 @@
 import React from 'react';
+import { Scrollbars } from 'react-custom-scrollbars';
+import { Controlled as CodeMirror } from 'react-codemirror2';
 
+import 'codemirror/lib/codemirror.css';
+
+import Title from './common/Title';
+import Input from './common/Input';
+
+import '../styles/NewSnippet.styl';
 
 const NewSnippet = () => (
-  <div>
-    New Snippet page
-  </div>
+  [
+    <Title title="New snippet" key="title" />,
+    <div className="new-snippet" key="new-snippet">
+      <div className="new-snippet-code-wrapper">
+        <div className="new-snippet-code-header">
+          <Input placeholder="Title" />
+          <Input placeholder="Tags (separate tags by comma)" />
+        </div>
+        <div className="new-snippet-code">
+          <CodeMirror
```

Comment:
Hmm, I gave it a try and looks like the code input is not editable. Is that intentional?

Reply:
Yeap, for now it is not editable as this is just stubbed data inside